RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -3603,6 +3603,7 @@ void auto_interruptCheck(boolean debug)
 	{
 		set_property("auto_interrupt", false);
 		restoreAllSettings();
+		auto_log_info("Autoscend thinks that you need " + meatreserve() + " meat for remaining quest requirements this ascension.");
```

Comment:
might want to only print this if > 0? Also possibly worth extracting the identical text used in multiple places to a single dedicated function.